src/tools/rbd_mirror/ImageReplayer.cc

```diff
@@ -842,14 +853,14 @@ void ImageReplayer<I>::handle_replay_ready()
 template <typename I>
 void ImageReplayer<I>::restart(Context *on_finish)
 {
```

Comment:
The original commit adds setting of `m_restart_requested` under `m_lock` here.  Why is it omitted?

Reply:
Oh, sorry. I lost this when resolving the conflict. Should be ok now.